proto/tendermint/abci/types.proto

```diff
@@ -285,6 +352,21 @@ message EventAttribute {
   bool   index = 3;  // nondeterministic
 }
 
+// ExecTxResult contains results of executing one individual transaction.
+//
+// * Its structure is equivalent to #ResponseDeliverTx which will be deprecated/deleted
+message ExecTxResult {
+  uint32         code       = 1;
+  bytes          data       = 2;
+  string         log        = 3;  // nondeterministic
+  string         info       = 4;  // nondeterministic
+  int64          gas_wanted = 5;
+  int64          gas_used   = 6;
+  repeated Event tx_events  = 7
+      [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];  // nondeterministic
+  string          codespace = 8;
+}
+
 // TxResult contains results of executing the transaction.
```

Comment:
Is this `TxResult` message used anywhere any more? Its `result` field is of type `ResponseDeliverTx`, which is deprecated right?

Reply:
Yeah! Well spotted :-). This message is not used elsewhere in the ABCI protos (neither ABCI not ABCI++), but it is used in some places in the code.

My impression is that we will need to through the whole codebase and try to remove all references to `TxResult`; then we can remove/deprecate the `TxResult` message here. But that was too much work for this PR. Hence I named the new message `ExecTxResult` to avoid conflicts.

If we manage to get rid of `TxResult` before v0.36, we might consider renaming `ExecTxResult` to `TxResult`. But this is not crucial.